File: Data Structures & Algorithms/sliding-window-maximum/submission-1.cpp
class Solution {
public:
    vector<int> maxSlidingWindow(vector<int>& nums, int k) {
        if (nums.empty()) return {};

  vector<int> res;
  for (int i = 0; i <= nums.size()-k; ++i)
  {
    int maxV= nums[i];
    for (int j = i; j < i+k; ++j)
    {
      maxV=max(maxV,nums[j]);
    }
    res.push_back(maxV);
  }
  return res;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.cs
public class Solution {
    public int[] TopKFrequent(int[] nums, int k) {
      Dictionary<int, int> map = new Dictionary<int, int>();
 int[] res = new int[k];
 for (global::System.Int32 i = 0; i < nums.Length; i++)
 {
     if (map.ContainsKey(nums[i]))
         map[nums[i]]++;
     else
         map.Add(nums[i], 1);
 }

 for (global::System.Int32 i = 0; i < k; i++)
 {
     int max = map.Values.Max();
     int index = map.Values.ToList().IndexOf(max);
     int target = map.Keys.ToList().ElementAt(index);
     res[i] = target;
     map.Remove(target);
 }


 return res;
    }
}
